genesyscloud/routing_settings/resource_genesyscloud_routing_settings_test.go

```diff
@@ -17,7 +17,7 @@ func TestAccResourceRoutingSettingsBasic(t *testing.T) {
 
 	resource.Test(t, resource.TestCase{
 		PreCheck:          func() { util.TestAccPreCheck(t) },
-		ProviderFactories: provider.GetProviderFactories(providerResources, providerDataSources),
+		ProviderFactories: provider.GetProviderFactories(providerResources, nil),
```

Comment:
providerDataSources are not required for this test case ?

Reply:
There is no data source for routing settings